transcriber/transcriber.py

```
@@ -0,0 +1,61 @@
+		print("Starting transcription...")
+		segments, info = model.transcribe(audio_path)
+
+		# Convert generator to list to get total segments
+		segments = list(segments)
+
+		# Write transcription to text file with progress bar
+		with open(output_file, "w", encoding="utf-8") as f:
+			for seg in tqdm(segments, desc="Transcribing", unit="segment"):
+				f.write(seg.text.strip() + "\n")
+
+		print(f"\nTranscription completed successfully!'")
```

Ruff (0.14.13)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)